docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-metrics-exporter.md

```diff
@@ -0,0 +1,99 @@
+---
+title: metrics-exporter resource
+headerTitle: ybm metrics-exporter
+linkTitle: metrics-exporter
+description: YugabyteDB Managed CLI reference metrics-exporter resource.
+headcontent: Manage metrics export configuration
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-cli-metrics-exporter
+    parent: managed-cli-reference
+    weight: 20
+type: docs
+---
+
+Use the `metrics-exporter` resource to create metrics export configurations for third-party tools, and assign them to clusters.
+
+## Syntax
+
+```text
+Usage: ybm metrics-exporter [command] [flags]
+```
+
+## Example
+
+Create a configuration:
+
+```sh
+ybm metrics-exporter create \
+    --config-name datadog1 \
+    --type DATADOG \
+    --datadog-spec api-key=efXXXXXXXXXXXXXXXXXXXXXXXXXXXXee,site=US1
+```
+
+## Commands
+
+### attach
+
+Assign an export configuration to the specified cluster.
+
+| Flag | Description |
+| :--- | :--- |
+| --config-name | Required. Name of the export configuration. |
+| --cluster-name | Required. Name of the cluster. |
+
+### create
+
+Create an export configuration.
+
+| Flag | Description |
+| :--- | :--- |
+| --config-name | Required. Name for the export configuration. |
+| --type | Required. The third party tool to exported metrics to. Options: DATADOG. |
+| --datadog-spec | Required for type DATADOG. The Datadog export details, provided as key-value pairs.<br>Arguments:<ul><li>api-key - your Datadog API key.</li><li>site - your datadog site parameters.</li></ul> |
+
+### delete
+
+Delete a specified export configuration. You can't delete configurations that are in use by a cluster.
+
+| Flag | Description |
+| :--- | :--- |
+| --config-name | Required. Name of the export configuration. |
```

Comment:
```suggestion
| --config-name | Required. Name for the export configuration. |
```

Reply:
No this is the name of the config to delete